### ISSUE TITLE
Enhance the charting tooltip design 

### ISSUE THREAD

**Issue by LHinson** [patternfly/patternfly-design#786]
Related issue/PR: 
PF React: https://github.com/patternfly/patternfly-react/issues/3219
Kiali (product): https://github.com/kiali/kiali-ui/pull/1457

Evaluate the charting tooltips and determine enhancements to improve the design. Leveraging @dlabrecq expertise with victory charting to understand what is feasible would be good.

**Comment by mcarrano** [comment]
@dlabrecq Is these feasible using Victory charts? What are the constraints we have if any?

**Comment by mcarrano** [comment]
Based on discussions with @dlabrecq and @nlcwong , looks like we require more investigation of what's possible using Victory charts before we can proceed forward on this. I've therefore reopened https://github.com/patternfly/patternfly-react/issues/3219 to track this. More detail on the design target can be found here: https://docs.google.com/document/d/1Z6GJHxuzsVZue54Yt2Ip_aRRXtXHrVlMr_h406v18ts/edit?usp=sharing  Further design work will be placed on hold pending further React investigation.

**Comment by mcarrano** [comment]
@dlabrecq @nlcwong I saw that some updates have been released in the PatternFly Charts package as of the latest release. Has this issue therefore been addressed or is there more to do here?

**Comment by nlcwong** [comment]
@mcarrano I think the only thing I have in mind is if we need to update the PF documentation section to include the tooltip enhancements or do we just leave it up to the react documentation to point out the fact that we now have a vertical bar and tooltip with chart legend type of information in.  Either way, it has been checked into the code as far as I understand things.

**Comment by dlabrecq** [comment]
The latest PatternFly charts package now includes a component to help display a tooltip legend. And if that does not fit future design needs, I also included a custom tooltip example which demonstrates how developers can create a custom tooltip using their own HTML / CSS layout in the chart's SVG.

+1 for the PF documentation. Although there is now a tooltip legend, there are also multiple tooltip variations. There are no guidelines with the React docs (i.e., when to use one over another).

**Comment by mcarrano** [comment]
Thanks @dlabrecq and @nlcwong . In that case I will close this design issue.

I have opened a documentation issue here and agree this should be added to our Charts usage guidelines: https://github.com/patternfly/patternfly-org/issues/1919  @nlcwong any interest in contributing that?

**Comment by nlcwong** [comment]
Sure!

NATALIE WONG

SENIOR UX INTERACTION DESIGNER

Red Hat <https://www.redhat.com/>

300 A Street, Boston, MA 02210

nwong@redhat.com    T: +1-857-770-0627    M: +1-508-963-6144
<https://red.ht/sig>
